src/platform/assets/utils/categoryLabel.ts

```diff
@@ -3,6 +3,9 @@ const ACRONYM_TAGS = new Set(['VAE', 'CLIP', 'GLIGEN'])
 export function formatCategoryLabel(raw?: string): string {
   if (!raw) return 'Models'
 
+  // Special display name mappings
+  if (raw === 'diffusion_models') return 'Diffusion'
```

Comment:
Not `'Diffusion Models'`?

I don't think of Diffusion as being a Noun in this context.